app/celery/tasks.py

```diff
@@ -697,19 +697,19 @@ def choose_sending_queue(process_type: str, notif_type: str, notifications_count
     queue: Optional[str] = process_type
 
     if notifications_count >= large_csv_threshold:
-        queue = QueueNames.BULK
+        queue = QueueNames.SEND_SMS_LOW if notif_type == SMS_TYPE else QueueNames.BULK
```

Comment:
can you also link the terraform pr that creates these queues? are we making them fifo as well?

Reply:
Ideally BULK/LOW queues would not be FIFO if we want to treat high volume for these, but we could check the numbers to see if we'd want to make it. So far, only db saving ops for high priority notifications are FIFO if I am not mistaken.

Reply:
currently the other queues used to send are not created in terraform but when they are used by api / celery, so I just copied that approach.